routing/router.go

```diff
@@ -1380,6 +1375,9 @@ func (r *ChannelRouter) FindRoute(source, target route.Vertex,
 		},
 		restrictions, source, target, amt,
 	)
+	if err != nil {
```

Comment:
arguably this could be put in its own PR that we can merge immediately :P

Reply:
there is no such thing as "merge immediately" in `lnd` :)

Reply:
😆